Compute/PollingAgent/src/ArmoniK.Core.Compute.PollingAgent.csproj

```diff
@@ -49,4 +49,7 @@
     <ProjectReference Include="..\..\..\Common\src\ArmoniK.Core.Common.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+	<AssemblyAttribute Include="System.Diagnostics.CodeAnalysis.ExcludeFromCodeCoverageAttribute" />
```

Comment:
Why is this (non-test) project is excluded from coverage?

Reply:
This project contains the entry point for a polling agent, main methods should not be tested but the methods it calls. Hence, I think they should be excluded from coverage.